packages/input/src/Input.js

```diff
@@ -37,34 +46,21 @@ function Input(props) {
         onBlur
       }) => (
         <Wrapper
-          isDisabled={props.disabled}
+          isDisabled={disabledProp}
           hasFocus={hasFocus}
           hasHover={hasHover}
           variant={props.variant}
         >
           <InputPresenter
-            autoComplete={props.autoComplete}
-            defaultValue={props.defaultValue}
-            disabled={props.disabled}
+            disabled={disabledProp}
             hasFocus={hasFocus}
             hasHover={hasHover}
-            id={props.id}
-            inputMode={props.inputMode}
-            maxLength={props.maxLength}
-            minLength={props.minLength}
-            name={props.name}
             onBlur={onBlur}
-            onChange={props.onChange}
             onFocus={onFocus}
             onMouseEnter={onMouseEnter}
             onMouseLeave={onMouseLeave}
-            pattern={props.pattern}
-            readOnly={props.readOnly}
-            required={props.required}
-            spellCheck={props.spellCheck}
-            tabIndex={props.tabIndex}
-            variant={props.variant}
-            value={props.value}
+            variant={variant}
+            {...otherProps}
```

Comment:
This is the heart of it. We gather any props we havn't used and pass them through.